scripts/README.md

```diff
@@ -7,6 +7,14 @@ Analytics service used to collect data to help the dYdX Product team make inform
 2. Add API key in Github > Secrets and Variables > Actions as `AMPLITUDE_API_KEY`
 3. In your deploy scripts add `pnpm run build:inject-amplitude` after your pnpm build / vite build command.
 
+### Hotjar
+User behavior tracking service used help the dYdX Product team make informed product decisions.
+
+<b>To use with dydxprotocol/v4-web:</b>
+1. Hotjar account with Site Id and Hotjar version.
+2. Add Site Id and Hotjar version in Github > Secrets and Variables > Actions as `HOTJAR_SITE_ID` and `HOTJAR_VERSION`
```

Comment:
ah the instruction for amplitude was outdated - we can keep this, but in reality the variables only need to be added on vercel

Reply:
confirming that DOS's prod deployment will have that in Vercel ✅